src/man/sssd.conf.5.xml

```diff
@@ -2062,6 +2062,29 @@ pam_gssapi_indicators_map = sudo:pkinit, sudo-i:pkinit
                         </para>
                     </listitem>
                 </varlistentry>
+                <varlistentry condition="build_json_pam">
+                    <term>pam_json_services (string)</term>
+                    <listitem>
+                        <para>
+                            Comma separated list of PAM services where the JSON
+                            protocol to communicate the available
+                            authentication mechanisms is enabled.
```

Comment:
Wording is a bit off IMO, maybe something like `Comma separated list of PAM services used to enable JSON authentication mechanism communication.`

or

`Comma separated list of PAM services used to communicate JSON authentication mechanism support.`

Reply:
Updated the text with the first option. Thanks!